orchagent/vnetorch.cpp

```diff
@@ -1066,18 +1346,48 @@ bool VNetRouteOrch::handleTunnel(const Request& request)
         }
     }
 
+    if (!vni_list.empty() && vni_list.size() != ip_list.size())
```

Comment:
I thought these conditions are required. The field should be either empty or the match in length. Otherwise, it would be impossible to match the fields with endpoints. For example, if the endpoint had three addresses, the vni field should be one of the choices: empty; `0,0,0`(three valid values); `0,,`(comma separated empty values). The field should not be something like `0,1` since it would be impossible to figure which endpoint it corresponds to.

I also noticed that the current implementation of request parser cannot handle the case of comma-separated empty values. Therefore, I updated the value type of mac and vni such to string so that we can tokenize the values and handle them in vnetorch.

Reply:
Added support for multiple endpoints to share the same vni in the latest commit.